identity/identity.go

```diff
@@ -164,6 +176,8 @@ func CredentialsFromEnv() *Credentials {
 			cred.UserDomain = defaultDomain
 		}
 	}
+	fmt.Sscanf(getConfig(CredEnvVersion), "%d", &cred.Version)
```

Comment:
There needs to be error checking that cred.Version is a number.
And we only want to do this if it is not ""